modules/navier_stokes/test/tests/finite_volume/pins/channel-flow/tests

```diff
@@ -0,0 +1,206 @@
+    method = "!dbg"
+    requirement = 'The system shall be able to model no-slip conditions in a porous media channel with reflective boundary conditions on one side; specifically, moving down the channel, the tangential velocity shall develop a parabolic profile.'
+    abs_zero = 1e-9
+    ad_indexing_type = 'global'
+    cli_args = "FVBCs/no-slip-u/boundary='top' FVBCs/no-slip-v/boundary='top' FVBCs/active='inlet-u inlet-v outlet-p no-slip-u no-slip-v symmetry-u symmetry-v symmetry-p' Outputs/file_base='rc-no-slip-symmetry'"
+  []
+  [symmetry-rz]
+    type = 'Exodiff'
+    input = 2d-rc.i
+    exodiff = rc-no-slip-rz-symmetry.e
+    method = "!dbg"
+    requirement = 'The system shall be able to model no-slip conditions in a cylindrical porous media channel with reflective boundary conditions on one side; specifically, moving down the channel, the tangential velocity shall develop a parabolic profile.'
+    abs_zero = 1e-9
+    ad_indexing_type = 'global'
+    cli_args = "Problem/coord_type='rz' Problem/rz_coord_axis='x' FVBCs/no-slip-u/boundary='top' FVBCs/no-slip-v/boundary='top' FVBCs/active='inlet-u inlet-v outlet-p no-slip-u no-slip-v symmetry-u symmetry-v symmetry-p' Outputs/file_base='rc-no-slip-rz-symmetry' -pc_type lu -pc_factor_shift_type NONZERO"
+  []
+  [friction]
+    type = 'Exodiff'
+    input = 2d-rc-friction.i
+    exodiff = rc-friction.e
+    method = "!dbg"
+    requirement = 'The system shall be able to model porous flow with volumetric friction, using the Darcy and Forchheimer friction models.'
+    abs_zero = 1e-9
+    ad_indexing_type = 'global'
+    cli_args = "FVBCs/active='inlet-u inlet-v outlet-p no-slip-u no-slip-v symmetry-u symmetry-v symmetry-p' Outputs/file_base='rc-friction' -pc_type lu -pc_factor_shift_type NONZERO"
+  []
+  [heated]
+    type = 'Exodiff'
+    input = 2d-rc-heated.i
+    exodiff = rc-heated.e
+    method = "!dbg"
+    abs_zero = 1e-9
+    requirement = 'The system shall be able to solve for fluid energy diffusion, advection and convection with the local medium in a 2D channel.'
+    ad_indexing_type = 'global'
+    design = 'PINSFVEnergyAdvection.md'
+    cli_args = "Outputs/file_base='rc-heated'"
+  []
+  [heated-rz]
+    type = 'Exodiff'
+    input = 2d-rc-heated.i
+    exodiff = rc-heated-rz.e
+    method = "!dbg"
+    abs_zero = 1e-9
+    requirement = 'The system shall be able to solve for fluid energy diffusion, advection and convection with the local medium in rz geometry.'
+    ad_indexing_type = 'global'
+    design = 'PINSFVEnergyAdvection.md'
+    cli_args = "Problem/coord_type='rz' Problem/rz_coord_axis='x' Outputs/file_base='rc-heated-rz'"
+  []
+  [heated-kappa]
+    type = 'Exodiff'
+    input = 2d-rc-heated-effective.i
+    exodiff = rc-heated-effective.e
+    method = "!dbg"
+    abs_zero = 1e-9
+    requirement = 'The system shall be able to solve for fluid energy effective diffusion, advection and convection with the local medium in a 2D channel.'
```

Comment:
Yea just unclear on what local medium is here. Is there a global medium? Is that the surroundings? If the distinction is our simulation domain is local then presumably the 2D channel is local too. I suggest clarifying any requirement in this `tests` spec that currently has `local medium`

Reply:
ok I ll replace local medium by solid phase